stablehlo/tests/bytecode.mlir

```diff
@@ -0,0 +1,769 @@
+// RUN: diff <(stablehlo-opt %s) <(stablehlo-opt -emit-bytecode %s | stablehlo-opt)
+// RUN: stablehlo-opt -emit-bytecode -debug-only=stablehlo-bytecode %s 2>&1 | (! grep 'Not Implemented')
```

Comment:
I'm not sure that debug output is available in non assert builds

Reply:
This shouldn't cause the test to error on non-asserted builds, unless `debug-only` on non-asserted errors. On non-asserted grep doesn't find "Not Implemented" anywhere and silently pass. But this still would catch issues regarding non-bytecoded constructs in CI runs.